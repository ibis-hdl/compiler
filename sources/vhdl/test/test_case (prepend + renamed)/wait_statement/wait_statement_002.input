-- VHDL test case input
wait on A,B; 
