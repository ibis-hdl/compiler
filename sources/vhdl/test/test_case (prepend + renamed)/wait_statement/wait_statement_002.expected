(wait_statement
  (sensitivity_clause
    (sensitivity_list
      (name<v>
        (identifier
          A
        )
      ),
      (name<v>
        (identifier
          B
        )
      )
    )
  )
)
